src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -253,4 +277,44 @@ public RegressionChecks generateTestChecks(ExecutableSequence sequence) {
     }
     return checks;
   }
+
+  /**
+   * Returns true if the method:
```

Comment:
A comment should first say what a method means or is intended to do.
This comment dives right into implementation details.  From that, it is difficult to understand the intention.

Once you write a proper specification for the method (which you should write based on what clients will find useful, not its current implementation), you will have a better feel regarding whether more tests, like a visibility test, should be folded into this method.